package-lock.json

```diff
@@ -9711,10 +9659,8 @@
 					}
 				},
 				"braces": {
-					"version": "3.0.2",
-					"resolved": "https://registry.npmjs.org/braces/-/braces-3.0.2.tgz",
```

Comment:
I don't know enough about the format of package-lock.json to understand what does it mean to not have a version and resolution. 

Just pointing it out in case someone else knows more.